views/user/profile.php

```diff
@@ -24,6 +24,16 @@
                 </div>
 
                 <ul>
+                    <?php if ($model->email): ?>
+                        <li>
+                            Your email is <?= Html::encode($model->email) ?>.
+                            <?php if ($model->email_verified): ?>
+                                It is verified.
+                            <?php else: ?>
+                                It is not verified yet. <?= Html::a('Verify', ['user/request-email-verification']) ?>.
+                            <?php endif ?>
+                        </li>
+                    <?php endif ?>
```

Comment:
if someone has logged in via oauth there is no need to verify email. This case should be covered.

Reply:
https://github.com/yiisoft-contrib/yiiframework.com/pull/222/commits/263b5556a28c3bc6adad514be244074369ef6aaa